controls/radoutlookbar/localization.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Localization
+page_title: Localization
+description: Localization
+slug: outlookbar-localization
+tags: localization
+published: True
+position: 3
+---
+
+# Localization
+
+__RadOutlookBar__ provides strings for localizing various of its elements. In order to be able to distinguish these resources, an unique identifier, called resource key, is assigned to each string that can be localized.
+
+Here is a list of all of the Resources available and their default values.
+
+Key	|	Value
+---	|	---	
+OutlookBarConfigureButtons | Configure buttons
+OutlookBarHorizontalSplitter | Start dragging in order to change items' position
+OutlookBarVerticalResizer | Start dragging in order to resize
+
+#### __Figure 1: Vertical Resizer__
+![Vertical Resizer](images/OutlookBar_Localization_01.png)
+
+#### __Figure 2: Horizontal Splitter__
+![Vertical Resizer](images/OutlookBar_Localization_02.png)
+
+#### __Figure 3: Configure Buttons__
+![Vertical Resizer](images/OutlookBar_Localization_03.png)
+
+## See also
+
+* [Localization]({%slug common-localization%})
```

Comment:
The article doesn't tell us why we need to know the resource keys. Move the link to the common Localization article in a note or a new paragraph with a short description on where you can use the keys.